docs/ibc/apps/bindports.md

```diff
@@ -0,0 +1,112 @@
+<!--
+order: 3
+-->
+
+# Bind ports
+
+Learn what changes to make to bind modules to their ports on initialization. {synopsis}
+
+## Pre-requisites Readings
+
+- [IBC Overview](../overview.md)) {prereq}
+- [IBC default integration](../integration.md) {prereq}
+
+Currently, ports must be bound on app initialization. In order to bind modules to their respective ports on initialization, the following needs to be implemented:
+
+1. Add port ID to the `GenesisState` proto definition:
```

Comment:
maybe clarify that in the case of SDK modules, the port ID is the name of the application (ID sometimes confuses people bc they think it should be numeric/iterative) and in the case of CosmWasm maybe a link to how CosmWasm contracts create and bind to custom ports?